frontend/stories/inputs/select/FormMultiSelect.stories.tsx

```diff
@@ -129,11 +129,11 @@ ControlledFormSelect.args = {
 export const ControlledFormSelectWithDefault = Controlled.bind({});
 ControlledFormSelectWithDefault.args = {
 	label: 'Controlled Multi Select input',
-	defaultValue: [4, 5],
+	defaultValue: ['4', '5'],
 };
 
 export const ControlledFormSelectWithDefaultDisabled = Controlled.bind({});
 ControlledFormSelectWithDefaultDisabled.args = {
```

Comment:
These two are almost the same. The second one doesn't have the default disabled

Reply:
Fair point